src/main/java/com/javamas/araignee/mq/MessageQueueManager.java

```diff
@@ -0,0 +1,96 @@
+package com.javamas.araignee.mq;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'com\.fasterxml\.jackson\.core\.JsonProcessingException' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.